benchmark/run.js

```diff
@@ -36,7 +36,8 @@ if (format === 'csv') {
 
 (function recursive(i) {
   const filename = benchmarks[i];
-  const child = fork(path.resolve(__dirname, filename), cli.optional.set);
+  const child = fork(path.resolve(__dirname, filename), cli.optional.set,
+                     { execArgv: ['--expose_internals'] });
```

Comment:
Don't you think it would be better to move the `require('internal/freelist')` inside main, instead of having all these `--expose_internals` around.